src/Sql/Sql/Replication/Cmdlet/NewAzureSqlDatabaseSecondary.cs

```diff
@@ -187,6 +195,7 @@ protected override IEnumerable<AzureReplicationLinkModel> ApplyUserInputToModel(
                 DatabaseName = this.DatabaseName,
                 PartnerResourceGroupName = this.PartnerResourceGroupName,
                 PartnerServerName = this.PartnerServerName,
+                PartnerDatabaseName = string.IsNullOrWhiteSpace(this.PartnerDatabaseName) ? this.DatabaseName : this.PartnerDatabaseName,
```

Comment:
What is the meaning of setting `PartnerDatabaseName` the same as `DataBaseName`?

Reply:
Hi.  This is current behavior.  For backwards compatibility, if client does not specify PartnerDatabaseName, they will get the same behavior as today which always uses DatabaseName for the partner.  A name is required no matter what.